vector/src/act/res/layout/vector_home_menu_header.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@drawable/act_gradient"
+    android:minHeight="@dimen/navigation_view_height">
+
+    <!--<ImageView
```

Comment:
Please explain why this is commented out

Reply:
It was originally commented on in that file. I made copies of this for our builds and left it as it. I made removed comments from files from our builds.